src/query/service/src/interpreters/interpreter_insert_v2.rs

```diff
@@ -101,6 +117,177 @@ impl InsertInterpreterV2 {
         let cast_needed = select_schema != *output_schema;
         Ok(cast_needed)
     }
+
+    fn apply_stage_options(
+        &self,
+        stage: &mut UserStageInfo,
+        params: &BTreeMap<String, String>,
+    ) -> Result<()> {
+        for (k, v) in params.iter() {
```

Comment:
we can separate copy options and format options.

and reuse fn like 

```
pub fn parse_copy_file_format_options(
    file_format_options: &BTreeMap<String, String>,
) -> Result<FileFormatOptions>
```

Reply:
Moved and separated this function to `src/meta/types/src/user_stage.rs`.
`parse_copy_file_format_options` maybe unified in next PR.